web/components/ui/PasswordStrength.tsx

```
@@ -0,0 +1,127 @@
+      case 3: return 'bg-yellow-500';
+      case 4: return 'bg-green-500';
+      default: return 'bg-gray-200 dark:bg-zinc-700';
+    }
+  };
+
+  const isNeo = style === THEMES.NEOBRUTALISM;
+
+  if (!password) return null;
+
+  return (
+    <div className="w-full flex flex-col gap-2 mt-2" aria-label="Password strength indicator">
```

Biome (2.3.13)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)